src/DnsHelperUI/MainForm.Designer.cs

```diff
@@ -236,13 +223,35 @@ private void InitializeComponent()
             this.btnChooseDns.UseVisualStyleBackColor = true;
             this.btnChooseDns.Click += new System.EventHandler(this.btnChooseDns_Click);
             // 
+            // cmbNics
+            // 
+            this.cmbNics.DropDownStyle = System.Windows.Forms.ComboBoxStyle.DropDownList;
```

Comment:
![image](https://cloud.githubusercontent.com/assets/6773890/19945519/844d03a2-a10e-11e6-8f24-3f18c64fbe6e.png)
Not a personal fan of this style, but its your application.